test/geoloco/Geoloco_Regression.ipynb

```diff
@@ -0,0 +1,452 @@
+    "                                           variables=mod08d3_geoloco_info['variable'],\n",
+    "                                           crs=utm_proj4_string)\n",
+    "\n",
+    "    mod021km_reprojection_compare_file = submit_and_download(harmony_client, mod021km_reprojection_request, file_indicators['MOD021KM'])\n",
+    "    mod35l2_reprojection_compare_file = submit_and_download(harmony_client, mod35l2_reprojection_request, file_indicators['MOD35_L2'])\n",
+    "    mod08d3_reprojection_compare_file = submit_and_download(harmony_client, mod08d3_reprojection_request, file_indicators['MOD08_D3'])\n",
+    "\n",
+    "    geoloco_reprojection_tests = 1\n",
+    "    \n",
+    "    mod021km_truth_dims = get_dim_sizes(reprojection_truth_data['MOD021KM'])\n",
+    "    mod021km_dims = get_dim_sizes(mod021km_reprojection_compare_file)\n",
+    "    if mod021km_truth_dims != mod021km_dims:\n",
+    "        print_error('MOD021KM Reprojection mismatch.')\n",
+    "        geoloco_reprojection_tests = 0\n",
+    "    \n",
+    "    mod35l2_truth_dims = get_dim_sizes(reprojection_truth_data['MOD35_L2'])\n",
+    "    mod35l2_dims = get_dim_sizes(mod35l2_reprojection_compare_file)\n",
+    "    if mod35l2_truth_dims != mod35l2_dims:\n",
+    "        print_error('MOD35_L2 Reprojection mismatch.')\n",
+    "        geoloco_reprojection_tests = 0\n",
+    "\n",
+    "    mod08d3_truth_dims = get_dim_sizes(reprojection_truth_data['MOD08_D3'])\n",
+    "    mod08d3_dims = get_dim_sizes(mod08d3_reprojection_compare_file)\n",
+    "    if mod08d3_truth_dims != mod08d3_dims:\n",
+    "        print_error('MOD08_D3 Reprojection file mismatch.')\n",
+    "        geoloco_reprojection_tests = 0\n",
+    "\n",
+    "    if geoloco_reprojection_tests == 1:\n",
+    "        print_success('Geoloco Reprojection requests.')\n",
+    "    \n",
+    "    remove_results_files()\n",
+    "else:\n",
+    "    print(f'Geoloco is not configured for this environment: \"{harnomy_environment}\" - skipping test.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1a711a55-8951-4530-a7fa-fc9c0f5e65d4",
+   "metadata": {},
+   "source": [
+    "## Regridding Regression Tests\n",
+    "\n",
+    "In the cell below, regridding is tested using a GEO `proj4` string for the Level 1B, Level 2 and Level 3 Collections/Granules and reprojection scale sizes. Outputs are considerably reduced using variable subsetting. Dimension sizes are checked between the truth data and the output data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d9898ae8-c1fa-4ba5-8b77-665db74e9914",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if (mod021km_geoloco_info is not None and \n",
+    "    mod35l2_geoloco_info is not None and\n",
+    "    mod08d3_geoloco_info is not None):\n",
+    "\n",
+    "    mod021km_regridding_request = Request(collection=mod021km_geoloco_info['collection'],\n",
```

Comment:
Right, I am talking to the team to Scott Longmore and Greg Ederer to try to speed it up considerably. Thinking about running them all at once with the corresponding reference_data generated from on prem.

Reply:
I sped up the tests tremendously by increasing the resolution, instead of decreasing the resolution and running reprojection, resampling and regridding in one iteration.